Sources/funnet/Core/NetworkResponder.swift

```diff
@@ -28,3 +18,31 @@ public struct NetworkResponder: NetworkResponderProtocol {
     
     public init() {}
 }
+
+public func stub<T: Codable>(_ responder: NetworkResponder, with model: T) -> (Fireable) -> Void {
```

Comment:
I like that we're moving these methods into the repo, but while we're doing that maybe we should think about whether `stub` can be interpreted by users and being immediate as opposed to a delay. 

Personally, I think there's an argument for having the `stub` method stub immediately and having a `stubWithDelay` function that does it with a delay

Reply:
Also, we may want to provide overloads so that we don't have to change how we stub all of our calls, something like 

```
stub<T: Codable>(_ call: NetworkCall, with model: T) -> (Fireable) -> Void {
   call.firingFunc = stub(call.responder, model)
}
```

Reply:
Agreed on withDelay – but as I tried adding the overloads it made things icky with the withDelay stuff. So, I'm going to just let us do a lazy refactor in any repos we decide to (since the existing ones still work).